packages/visibility_detector/example/lib/main.dart

```diff
@@ -74,6 +74,7 @@ class VisibilityDetectorDemo extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       title: title,
+      scrollBehavior: const MaterialScrollBehavior().copyWith(scrollbars: false),
```

Comment:
is this related?

Reply:
This makes the framework stop complaining when you run the example app in desktop mode. It's not essential for this change.

Reply:
(No, it's not related)